.github/workflows/pull-request.yml

```diff
@@ -77,6 +77,12 @@ jobs:
           kubectl logs -n konveyor-forklift $(kubectl get po -n konveyor-forklift  -o=name | grep forklift-controller) >> /tmp/artifacts/k8s-forklift-controller-inventory.log
           kubectl get Storageclasses -A -o yaml >> /tmp/artifacts/k8s-storage-classes.log
           
+          if [ "${{ matrix.source_provider }}" == 'ovirt' ] ; then
```

Comment:
@eslutsky why only to ovirt?

Reply:
fixed

Reply:
@eslutsky i saved all the changes to a single PR locally, let me know if you want me to post it or you will.